treeherder/etl/perf_data_adapters.py

```diff
@@ -235,6 +234,48 @@ def adapt_and_load(self, reference_data, job_data, datum):
             result_set_id = job_data[_job_guid]['result_set_id']
             push_timestamp = job_data[_job_guid]['push_timestamp']
 
+            # counters will not be part of the summary series
+            # counters have a json obj {'stat': val} instead of [val1, val2, ...]
+            for _test in talos_datum["talos_counters"].keys():
```

Comment:
It looks like you're assuming that the talos object always has a talos_counters key, which is not necessarily the case. This is causing the unit tests to fail.

That brings up the point that (1) the unit test sample data should be updated to incorporate some counters, so we're testing that and (2) we should only process talos_counters if that data actually exists